.github/release.yml

```diff
@@ -0,0 +1,35 @@
+changelog:
+
+  exclude:
+    authors:
+      - dependabot
+      - dependabot[bot]
+
+  categories:
+    - title: Security Fixes
+      labels:
+        - C-security
+
+    - title: New Features
+      labels:
+        - C-new-feature
+
+    - title: Feature Enhancements
+      labels:
+        - C-enhancement
+
+    - title: Usability and Accessibility
+      labels:
+        - "A-*"
```

Comment:
That's not quite true. That `A-` labels just specify what area of studio the issue is concerned with.

But regarding this whole label mapping: the current labels are a bit of a mess and maybe not properly used. Also, they were designed for issues, not pull requests. We can easily rename some or add new ones. 

I would suggest that we select a small number of labels that we intend to assign to pull requests. That's what we did in Tobira as well. Most of the things you list here already make sense though: `C-security`, `C-new-feature`, `bug`, `documentation`. My worry is that some of those start with `C-` and others don't. So it's unclear which ones are the magic ones that need to be assigned to PRs. In Tobira we have 5 specific labels for PRs, all starting with `changelog:`. That makes the process of selecting labels easier. 

So rather than looking at what labels already exist, the question is: what sections/categories would you, ideally, want in the changelog? For comparison, in Tobira we have the sections "user facing changes", "primarily for admins" and "primarily for developers". Does that make sense for Studio? Or should Studio rather go with a more traditional bugfix/feature/... approach? What do you think?

Reply:
I would go with more traditional labels and also like the idea of ​​giving labels a start tag.

I would suggest the following labels and add them if you agree.
- changelog:security
- changelog:new-feature
- changelog:feature-improvement
- changelog:accessibility
- changelog:bug-fix

Do I have to document/announce this somewhere so that others know about it? And of course use these new labels for PRs.